content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,262 @@
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/pipelines/overview/pipelines-overview/">Pipelines</a>
+          (<a href="https://github.com/kubeflow/pipelines">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>0.1.40</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/other-guides/multi-user-overview/">Profile 
+          Controller for multi-user isolation</a> 
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/pytorch/">PyTorch training: PyTorch operator</a> 
+          (<a href="https://github.com/kubeflow/pytorch-operator">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/tfbatchpredict/">TensorFlow 
+          batch prediction</a>
+          (<a href="https://github.com/kubeflow/batch-predict">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/tfserving_new/">TensorFlow
+          serving</a></td>
+        <td>Stable</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/tftraining/">TensorFlow training:
+          TFJob operator</a>
+          (<a href="https://github.com/kubeflow/tf-operator">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td>XGBoost training: XGBoost operator
+        (<a href="https://github.com/kubeflow/xgboost-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<a id="sdk-matrix"></a>
+## Kubeflow SDKs and CLIs
+
+Alongside Kubeflow {{% kf-latest-version %}}, you may want to use 
+one of the following Kubeflow SDKs and command-line interfaces 
+(CLIs).
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>SDK / CLI</th>
+        <th>Status with Kubeflow {{% kf-latest-version %}}</th>
+        <th>SDK/CLI version</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="/docs/fairing/fairing-overview/">Fairing</a> 
+          (<a href="https://github.com/kubeflow/fairing">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/other-guides/kustomize/">kfctl</a> 
+          (<a href="https://github.com/kubeflow/kfctl">GitHub</a> )
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/pipelines/sdk/sdk-overview/">Kubeflow Pipelines SDK</a> 
+          (<a href="https://github.com/kubeflow/pipelines">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>0.1.40</td>
```

Comment:
/cc @IronPan corresponding SDK verison.

Reply:
We are shipping pipelines 0.2.0
https://github.com/kubeflow/manifests/blob/v1.0-branch/pipeline/pipelines-ui/base/kustomization.yaml#L15

Reply:
@IronPan I've changed the SDK version to **0.2.0** but we still need confirmation that this is the correct number.

Reply:
I have confirmation on chat from IronPan that the SDK version is also 0.2.0. The backend and SDK versions are mapped 1:1.